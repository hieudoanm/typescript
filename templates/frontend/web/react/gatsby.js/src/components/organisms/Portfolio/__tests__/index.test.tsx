import { render } from '@testing-library/react';
import { Portfolio } from '../index';
import React from 'react';

describe('Portfolio', () => {
  it('renders correctly', () => {
    const button = render(
      <Portfolio title={''} description={''} projects={[]} />
    );
    expect(button.container).toMatchSnapshot();
  });
});
