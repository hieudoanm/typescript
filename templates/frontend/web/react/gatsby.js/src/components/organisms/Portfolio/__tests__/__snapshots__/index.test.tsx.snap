// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio renders correctly 1`] = `
<div>
  <section
    class="py-8 md:py-16"
    id="projects"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="mb-8 md:mb-16"
      >
        <div
          class="text-center mx-auto max-w-lg"
        >
          <h1
            class="text-xl sm:text-3xl font-b162 mb-4 font-bold"
          />
          <h2
            class="text-sm sm:text-lg"
          />
        </div>
      </div>
      <div
        class="grid grid-cols-1 gap-8 md:grid-cols-3"
      />
    </div>
  </section>
</div>
`;
